backend/index.js

```diff
@@ -0,0 +1,65 @@
+const express = require('express')
+const app = express()
+const mongoose = require('mongoose')
+const Recipe = require('./models/recipeSchema')
+const connection_url = "mongodb+srv://dbMeganTrieu:YHb5aXyiQPomvX4Q@cluster0.rtwvj.mongodb.net/RecipesDB?retryWrites=true&w=majority"
+
+mongoose.connect(connection_url)
+    .then(() => console.log('Successfully connected'))
+    .catch((error) => console.error(`Could not connect due to ${error}`))
+    
+app.use(express.json())
+app.get('/', (req, res) => {
+  res.send('Hello world!')
+})
+app.listen(3001)
```

Comment:
not a big deal, but it is a best practice to put this line at the bottom of the file.